tests/models/mistral/test_modeling_mistral.py

```diff
@@ -729,15 +730,24 @@ def test_compile_static_cache(self):
 @slow
 @require_torch_gpu
 class Mask4DTestHard(unittest.TestCase):
+    model_name = "mistralai/Mistral-7B-v0.1"
+    _model = None
+
     def tearDown(self):
         gc.collect()
         torch.cuda.empty_cache()
 
+    @cached_property
+    def model(self):
+        if self.__class__._model is None:
+            self.__class__._model = MistralForCausalLM.from_pretrained(
+                self.model_name, torch_dtype=self.model_dtype
+            ).to(torch_device)
+        return self.__class__._model
```

Comment:
This is not ideal, but since it's slow tests which is meant to be run in a single process, we can accept this somehow hacky solution to avoid GPU OOM.

Reply:
Do we even need `@cached_property` here?

Reply:
No :-), you have a very good 👁️ ! I will remove it.
(Actually, after this PR, I started to doubt if `cached_property` is working well with tests!)